plugins/user-settings-backend/src/service/router.test.ts

```diff
@@ -50,7 +52,12 @@ describe('createRouter', () => {
       signals: signalService as SignalsService,
     });
 
-    app = express().use(router);
+    const errorHandler = MiddlewareFactory.create({
+      config: mockServices.rootConfig(),
+      logger: mockServices.rootLogger(),
+    }).error();
```

Comment:
It could, but I was wondering if there are a way of testing it without having to mock the errorHandler manually, is that possible using the [startTestBackend](https://backstage.io/docs/backend-system/building-plugins-and-modules/testing)?

Reply:
I extracted it to the `test-utils` package